packages/solid/store/src/mutable.ts

```diff
@@ -42,7 +42,7 @@ const proxyTraps: ProxyHandler<StoreNode> = {
   has(target, property) {
     if (property === $RAW || property === $PROXY || property === $TRACK ||
         property === $NODE || property === "__proto__") return true;
-    getDataNodes(target)[property]?(); // track
+    getDataNodes(target)[property]?.(); // track
```

Comment:
In general, I've avoid optional chaining in the code base as we don't do any transforms on the code base and in some environments it die. Every time one slips in something breaks something. As annoying as it is, that's why all the others are written the way they are.

Reply:
Oops, right; I forgot.